swaglyrics_backend/issue_maker.py

```diff
@@ -52,12 +52,8 @@
 # artist and song regex
 asrg = re.compile(r'[A-Za-z\s]+')
 
-SQLALCHEMY_DATABASE_URI = "mysql+mysqlconnector://{username}:{password}@{username}.mysql.pythonanywhere-services." \
-                          "com/{username}${databasename}".format(
-                                                                username=username,
-                                                                password=os.environ['DB_PWD'],
-                                                                databasename="strippers"
-                                                            )
+SQLALCHEMY_DATABASE_URI = f"mysql+mysqlconnector://{username}:{os.environ['DB_PWD']}@{username}.mysql" \
```

Comment:
I'd actually prefer to keep the old formatting strings here since that is more readable and coherent